### ISSUE TITLE
Calendar not syncing

### ISSUE THREAD

**Issue by dabathehutt** [valentine195/obsidian-fantasy-calendar#4]
Having an issue that a created calendar isn't synced via Obsidian Sync. Is this a known issue already? Absolutely fantastic work, besides that!

**Comment by valentine195** [comment]
Are you syncing the plugin folder in `.obsidian`? The data is stored in there.

**Comment by dabathehutt** [comment]
Yes, I am syncing everything that's possible to choose in the Obsidian Sync menu. Just had a look into the .obsidian > plugins > fantasy-calendar folder. But there are only 4 files (data.json, main.js, manifest.json and styles.css) - all of them not storing the calendar data, I guess. So I am not sure where the calendar data is saved :/

**Comment by valentine195** [comment]
The data will be in the `data.json` file. If that file is being synced, the calendars will sync.

**Comment by dabathehutt** [comment]
Okay, weird - because I can open and close obsidian - the plugin "remembers" the data. But my data.json file looks pretty empty:

```
{
  "calendars": [],
  "currentCalendar": null,
  "defaultCalendar": null,
  "eventPreview": false,
  "path": "/"
}
```

**Comment by valentine195** [comment]
Well, let me rephrase - the plugin will create a `data.json` file in that folder no matter what. The *content* inside needs to be synced between devices, as the plugin reads the data file when it loads to load all of its saved data.

I have not used sync, but it could be that the plugin had created an empty data file here before it had a chance to sync the data file from the other device, and now it won't sync from the other device.

Make a backup of your original data file, and maybe try deleting the empty one and seeing if it syncs then. If not, try copying and pasting the original data file into the other device's data folder?

**Comment by dabathehutt** [comment]
Seems like this was an error on the Sync site! I removed the vault completely on all of my devices and set it up again for all of them and now everything works fine. Don't know where the issue was in detail though. Thank you for your great support anyway!
